src/routes/index.js

```diff
@@ -1,14 +1,42 @@
-import { Auth } from "../modules/auth/pages";
-import { Main } from "../modules/main/pages";
+import Auth from "../modules/auth/index";
```

Comment:
You can remove `index` from all of these paths. Entry points (`index`) is transparent and read as a first file in the path if the certain path hasn't been pointed.